.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       xcode: "9.0"
     steps:
       - checkout
-      - run: HOMEBREW_NO_AUTO_UPDATE=1 brew install awscli
+      - run: HOMEBREW_NO_AUTO_UPDATE=1 brew install awscli openssl
```

Comment:
Wonder if we need to `apt-get install libssl-dev` for the Linux builds as well...